src/api/goods.ts

```diff
@@ -3,17 +3,48 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
+const request = async (url: string) => {
+  try {
+    const response = await fetch(url);
+
+    return await response.json();
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (error: any) {
+    // eslint-disable-next-line no-console
+    console.log('error', error.message);
+
+    return error;
+  }
+};
+
 export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+  const answerFromServer = async () => {
+    const promise = await request(API_URL);
+
+    return promise;
+  };
+
+  return answerFromServer();
 }
 
 export const get5First = () => {
-  return getAll()
-    .then(goods => goods); // sort and get the first 5
+  const answerFromServer = async () => {
+    const jsonData = await request(API_URL);
```

Comment:
why json data? you converted json on js object via json() function

Reply:
i will fix it